crates/fluvio-cluster/src/error.rs

```diff
@@ -162,6 +162,12 @@ pub enum LocalInstallError {
     ClusterCheckError(#[from] ClusterCheckError),
     #[error("Progress Error: {0}")]
     ProgressError(#[from] TemplateError),
+    /// Timed out when waiting for etcd.
```

Comment:
Need to migrate to anyhow.  doesn't need for this

Reply:
It should be migrated all together. For now, it is better to keep the same style.